crates/dojo-erc/src/erc20/systems.cairo

```diff
@@ -17,6 +17,52 @@ mod erc20_approve {
     }
 }
 
+#[system]
+mod erc20_increase_allowance {
+    use starknet::ContractAddress;
+    use zeroable::Zeroable;
+
+    use dojo_erc::erc20::components::Allowance;
+    use dojo::world::Context;
+
+    fn execute(
+        ctx: Context,
+        token: ContractAddress,
+        owner: ContractAddress,
+        spender: ContractAddress,
+        added_value: felt252
+    ) {
+        let current_allowance = get!(ctx.world, (token, owner, spender), Allowance);
+        set!(
+            ctx.world,
+            Allowance { token, owner, spender, amount: current_allowance.amount + added_value }
+        )
```

Comment:
And I think these should be wrapped as a method / free function on the Allowance component itself.

Furthermore since the allowance is felt252 you have an overflow / underflow issue I think ? If you add the checks, it makes more sense to wrap this in a function for potential other callers.